.github/workflows/reusable_codespell.yml

```diff
@@ -0,0 +1,38 @@
+name: Codespell
```

Comment:
I believe you can use `reusable_checks.yml` workflow and just add an extra step next to existing `Run a spell check`

Reply:
Script might be useful in case in someone want to use it from their development setup. So instead looking to yml, and copying command, they can just call script which has all correct configuration. 

TBH the best solution would be to put this checker in to cmake so we have additional target, similar to cstyle, but script is still beter then yml